cfme/tests/cloud_insights/test_cloud_insights_manual.py

```diff
@@ -0,0 +1,119 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+from cfme import test_requirements
+
+pytestmark = [
+    pytest.mark.tier(0),
+    test_requirements.access
+]
+
+
+@pytest.mark.manual
```

Comment:
_Optional_ I would put this in the `pytestmark` since this is a manual tests only file.